b9lab-content/5-my-own-chain/03-starport.md

```diff
@@ -1 +1,210 @@
-# Starport
\ No newline at end of file
+# Starport
+
+## Install
+
+The quickest and simplest way to create blockchain applications with the Cosmos SDK is to use Starport. 
+
+To install Starport at the command line:
+
+```bash
+$ curl https://get.starport.network/starport! | bash
+
```

Comment:
Learn more about how to [install Starport](https://docs.starport.com/guide/install.html).

Reply:
I included a change with a direct in a highlight box.